src/components/EditionCard/EditionCard.tsx

```diff
@@ -1,16 +1,21 @@
 import React from "react";
 import * as DS from "@nypl/design-system-react-components";
 import Link from "../Link/Link";
-import { WorkEdition } from "~/src/types/DataModel";
+import { ApiItem, WorkEdition } from "~/src/types/DataModel";
 import EditionCardUtils from "~/src/util/EditionCardUtils";
 import { PLACEHOLDER_COVER_LINK } from "~/src/constants/editioncard";
+import { useCookies } from "react-cookie";
+import { MediaTypes } from "~/src/constants/mediaTypes";
+
+export const NYPL_SESSION_ID = "nyplIdentityPatron";
 
 export const EditionCard: React.FC<{ edition: WorkEdition; title: string }> = ({
   edition,
   title,
 }) => {
+  const [cookies] = useCookies([NYPL_SESSION_ID]);
+
```

Comment:
The example from react-cookie wraps the whole app with CookiesProvider, but it seems to work without it in our case, maybe because that's only used for setting cookies?

Reply:
yes, I think that's correct.